src/js/components/controls/NumericSlider.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import '../../../scss/components/NumericSlider.scss'
+
+class NumericSlider extends Component{
+    constructor(props){
+        super(props);
+        this.props = props;
+    }
+
+    render() {
+        return <input id={this.props.setting.key} name={this.props.setting.key}
+            type="range" min="10" max="1000" defaultValue={this.props.setting.value} className="slider"
```

Comment:
I would pass the min and max values with the props, So we can use this component for other settings.
This min and max are specific for the scroll speed

Reply:
If you did this (which I see you did), change the names of the constants inside `NumericSlider` from `MIN` and `MAX` to `DEFAULT_MIN` and `DEFAULT_MAX`.

Reply:
Done